### PR TITLE
Admin endpoint docs

### PR DESCRIPTION
**Pull request by samrabelachew**
Added documentation on current state of Admin API and created a docs section for future use as xds-relay grows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,3 +125,6 @@ Sample result:
 This is a gif of a tmux session demonstrating this example:
 
 ![demo](example/xds-relay-demo.gif)
+
+## Docs
+For more details on how to use the xDS Aggregator, see documentation [here](docs)
```
Comment (jessicayuen) [inline README.md:130]:
```suggestion
For more details on how to use and observe xd-relay, see documentation [here](docs)
```

File: docs/AdminEndpoints.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Admin Endpoints
```
Comment (jessicayuen) [inline docs/AdminEndpoints.md:1]: This detail looks great. What do you think of structuring it into this format:

```
## `GET /cache/<key>`

### Behavior
Searches through the xds-relay cache and outputs the contents of the cached xDS response for the given aggregated key.

### Parameters
*`<key>`*: The aggregated key or prefix to be searched.
  * Argument supports wildcard (*) glob suffix patterns 
  * ...

### Example
...

**Review by jessicayuen — COMMENTED** [review]
Thanks for taking the initiative to do this! Couple comments:

File: docs/AdminEndpoints.md (hunk 2 above)
Comment (samrabelachew) [reply docs/AdminEndpoints.md:1]: Added this structure, but made example outputs brief to keep file short

**Pull request closed without merging**